### PR TITLE
enable omniscidb-common on windows

### PR DESCRIPTION
**Pull request by guilhermeleobas**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by guilhermeleobas** [comment]
@conda-forge-admin, please restart ci

**Comment by guilhermeleobas** [comment]
@conda-forge-admin, please restart ci

File: recipe/meta.yaml [hunk 1]
```diff
@@ -77,53 +80,77 @@ outputs:
         # so that one could run cmake for generating *.bc, and other
         # common data files.
         - arrow-cpp ={{ arrow_version }}=*cpu
-        - bisonpp
+        - bisonpp  # [linux64]
```
Comment (pearu) [inline recipe/meta.yaml:83]:
```suggestion
        - bisonpp  # [not win]
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -77,53 +80,77 @@ outputs:
...
         - blosc
         - boost-cpp
         - clangdev {{ llvm_version }}
         - double-conversion
-        - flex
+        - flex  # [linux64]
```
Comment (pearu) [inline recipe/meta.yaml:88]:
```suggestion
        - flex  # [not win]
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -77,53 +80,77 @@ outputs:
...
         - glog
         - llvmdev {{ llvm_version }}
         - llvm {{ llvm_version }}
         - libarchive
+        - libevent  # [win]
         - libgdal
         - libpng
         - librdkafka
+        - tbb
+        - tbb-devel
```
Comment (pearu) [inline recipe/meta.yaml:100]: Is tbb and tbb-devel needed just for windows?

File: recipe/meta.yaml [hunk 4]
```diff
@@ -77,53 +80,77 @@ outputs:
...
         - thrift-cpp {{ thrift_version }}
+        - winflexbison  # [win]
     test:
       commands:
         # Test installation
         # doc
-        - test -f ${PREFIX}/share/doc/omnisci/LICENSE.md
+        - test -f ${PREFIX}/share/doc/omnisci/LICENSE.md  # [linux64]
```
Comment (pearu) [inline recipe/meta.yaml:107]:
```suggestion
        - test -f ${PREFIX}/share/doc/omnisci/LICENSE.md  # [not win]
```
The same applies to all changes below in order to support osx target as well.

File: recipe/meta.yaml [hunk 5]
```diff
@@ -77,53 +80,77 @@ outputs:
...
+        - if not exist %PREFIX%\bin\omnisci-utility-{{ base_version }}.jar exit 1  # [win]
+        - if not exist %PREFIX%\bin\omnisci-jdbc-{{ base_version }}.jar exit 1  # [win]
+        - if not exist %PREFIX%\bin\calcite-1.0-SNAPSHOT-jar-with-dependencies.jar exit 1  # [win]
         # Unspecified
-        - test -f ${PREFIX}/{{ install_base }}/bin/startomnisci
-        - test -f ${PREFIX}/{{ install_base }}/bin/omnisci_insert_sample_data
+        - test -f ${PREFIX}/{{ install_base }}/bin/startomnisci  # [linux64]
+        - test -f ${PREFIX}/{{ install_base }}/bin/omnisci_insert_sample_data  # [linux64]
+        # - if not exist %PREFIX%\bin\startomnisci.exe exit 1  # [win]
+        # - if not exist %PREFIX%\bin\omnisci_insert_sample_data.exe exit 1  # [win]
```
Comment (pearu) [inline recipe/meta.yaml:150]: `startomnisci` and `omnisci_insert_sample_data` are `bash` scripts. I guess OmniSci should provide the corresponding `.bat` files.

**Review by pearu — CHANGES_REQUESTED** [review]
@guilhermeleobas I did a quick review: let's not forget the future osx support, that is, instead of `[linux64]` use `[not win]` as it allows fewer changes to the recipe when adding osx support.

File: recipe/meta.yaml (hunk 3 above)
Comment (guilhermeleobas) [reply recipe/meta.yaml:100]: Yes! The linux build works just fine.

File: recipe/meta.yaml [hunk 6]
```diff
@@ -167,12 +193,12 @@ outputs:
         - maven
       host:
         - arrow-cpp ={{ arrow_version }}=*{{ build_ext }}
-        - bisonpp  # [not win]
+        - bisonpp  # [linux64]
```
Comment (pearu) [inline recipe/meta.yaml:196]:
```suggestion
        - bisonpp  # [unix]
```
Same applies below as well.

**Review by pearu — APPROVED** [review]
Nit: conda build on windows shows a warning:
```
cl : Command line warning D9002 : ignoring unknown option '/w0'
```
According to https://docs.microsoft.com/en-us/cpp/build/reference/compiler-option-warning-level?view=msvc-170, the option should be `/w` or `/W0`

Otherwise, LGTM! Thanks, @guilhermeleobas !

**Pull request merged**